Makefile

```diff
@@ -158,8 +158,12 @@ test: build ## run the unit, integration and docker-py tests
 test-docker-py: build ## run the docker-py tests
 	$(DOCKER_RUN_DOCKER) hack/make.sh dynbinary test-docker-py
 
+# Deprecated: use `test-integration` instead
```

Comment:
Can you add the "deprecation" message in the comment below? the `## run the integration tests` comment is also used for outputting the help message (`make help`)